ansible/roles/common_installs/tasks/main.yml

```diff
@@ -92,7 +93,6 @@
   pip: name={{ item }}
   sudo: yes
   with_items:
-    - fabric
```

Comment:
Yeah I'm really not sure. I traced it back all the way to the first commit of the repo https://github.com/dimagi/commcarehq-ansible/commit/699d5d945b706db23a55327b8c20d668b259ef3c#diff-4d74503b386f59bf023b93f3dd8d5a3cR18, that's been moved around a dozen times to wind up where it is now, and was originally added by someone we very randomly contracted. My guess is that if you move it, nothing will break. It certainly wasn't added there for a specific purpose by a dimagi employee recently (or ever).